tools/truncate-silence.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -e
+tools_path=$(dirname "$0")
+
+usage() {
+    echo "Usage: $0 [OPTION] [FILE]" 1>&2;
+    echo "Truncates silence from the start or the end of an audio file"
+    echo "    -c    Check ffmpeg." 1>&2;
+    echo "    -s    Truncate files from the start of the stream." 1>&2;
```

Comment:
I think you meant 'truncate silence' here?

Reply:
yes